fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/view/SortComparison.java

```diff
@@ -77,4 +80,9 @@ public String typelessString() {
     public int planHash(@Nonnull final PlanHashKind hashKind) {
         return 0;
```

Comment:
Again, I think this should have your planHash logic with the `BASE_HASH` in it, correct?

Reply:
Of course. Done.